yt_dlp/utils.py

```diff
@@ -1386,7 +1385,10 @@ def http_request(self, req):
         if url != url_escaped:
             req = update_Request(req, url=url_escaped)
 
-        for h, v in self._params.get('http_headers', std_headers).items():
+        headers = merge_headers(
+            {'Accept-Encoding': ', '.join(SUPPORTED_ENCODINGS)}, self._params.get('http_headers', std_headers))
+
+        for h, v in headers.items():
```

Comment:
isn't it better to do:
```py
for h, v in headers.items():
    ...
if 'Accept-Encoding' not in req.headers:
    req.add_header('Accept-Encoding', ', '.join(SUPPORTED_ENCODINGS))
```
no need to pass it though merge_headers

Reply:
Yeah true. that looks cleaner too :sweat_smile: